src/ios/netcdf_updater.F90

```diff
@@ -0,0 +1,196 @@
+! Copyright (C) 2020 National Center for Atmospheric Research
+! SPDX-License-Identifier: Apache-2.0
+!
+!> \file
+!> The musica_netcdf_updater module
+
+!> The netcdf_updater_t type and related functions
```

Comment:
I’d like a more motivated description of this type

Reply:
I've been putting the more detailed descriptions above the type definition:
```
  !> Updater for a paired MUSICA <-> NetCDF variable
  !!
  !! Staging data and functions for updating MUSICA state variables from input
  !! data and updating output files from the MUSICA state.
  !!
  type :: netcdf_updater_t
```
Is this sufficient? I'm going to add tests and example usage for all the file (and other) types that don't currently have them after the last use case is done - just because some of these types will have to change (like the NetCDF dimensions, e.g.) to handle the remaining use cases.